src/components/Login.js

```diff
@@ -120,8 +119,8 @@ class Login extends Component {
                                 </div>
                               </div>
                             </div>
-                            <div className="footer text-center">
-                              <button onClick={this.submitForm} className="btn btn-info btn-round btn-wd btn-lg">Let's go</button>
+                            <div className="footer text-center" style={{display:'flex', alignItems: 'center', justifyContent: 'center'}}>
```

Comment:
why not add the CSS in the CSS file?

Reply:
Thanks for point out. I will put that into my CSS file.

Reply:
Just fixed this in my latest commit